GovUkDesignSystem/GovUkHtmlHelperExtensions.cs

```diff
@@ -343,6 +343,7 @@ public static class GovUkHtmlHelperExtensions
             Dictionary<TEnum, Conditional> conditionalOptions = null,
             Dictionary<TEnum, LabelViewModel> labelOptions = null,
             Dictionary<TEnum, Dictionary<string,string>> attributeOptions = null,
+            ICollection<TEnum> radioOptions = null,
```

Comment:
I think `overrideRadioValues` would be a bit clearer than `radioOptions`. Also can we make it IEnumerable?